typescript/packages/lookslike-high-level/src/components/window-manager.ts

```diff
@@ -127,19 +133,9 @@ export class CommonWindowManager extends LitElement {
     });
   }
 
-  onClose(e: Event) {
-    const id = (e.currentTarget as HTMLElement).parentElement?.id;
-    if (id) {
-      const idCounts: { [key: string]: number } = {};
-
-      this.sagas = this.sagas.filter((saga) => {
-        const sagaId = saga.getAsProxy()[ID];
-
-        idCounts[sagaId] ??= 0;
-        const sagaID = sagaId + "#" + idCounts[sagaId]++;
-        return sagaID !== id;
-      });
-    }
+  onClose(e: Event, windowId: string) {
```

Comment:
@seefeldb although I do like that close and render have direct access to the windowId without having to do a lot of work... 

I DO NOT KNOW WHAT WE ARE GIVING AWAY BY DO THING THIS THOUGH